app/views/items/destroy.js.erb

```diff
@@ -0,0 +1,5 @@
+<% if @item.destroyed? %>
+  $('#item-<%= @item.id %>').fadeOut( "slow" );
+<% else %>
+  $('#item-<%= @item.id %>').prepend("<div class='alert alert-danger'><%= flash[:error] %></div>");
```

Comment:
In case you're seeing the same flash message appear more than once, consider using `flash.discard(...)`, everywhere flash messages are output. E.g. in this file, it'd become:

```html
...<%= flash.discard(:error) %>...
```